src/Altinn.Notifications.Core/Altinn.Notifications.Core.csproj

```diff
@@ -7,8 +7,23 @@
     <ProjectGuid>{607C02FB-8C76-41BD-A3DF-282B2D0E2296}</ProjectGuid>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.1" />
+<ItemGroup Condition="'$(Configuration)'=='Debug'">
+    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.435">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <AdditionalFiles Include="..\..\stylecop.json">
+      <Link>stylecop.json</Link>
+    </AdditionalFiles>
   </ItemGroup>
 
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>$(NoWarn);1591</NoWarn>
```

Comment:
Probably don't need the NoWarn tag. We want XML comment on public visible types and members.
`1591: Missing XML comment for publicly visible type or member 'Type_or_Member'`